src/auth/auth.proto

```diff
@@ -328,11 +333,17 @@ message SessionInfo {
   // information to a user who has network access to Pachyderm but not an
   // account in the OIDC provider.
   bool conversion_err = 3;
+  // rawIdToken is the ID token returned by the OAuth2 Flow when exchanged with the user's auth_code
+  string rawIdToken = 4;
```

Comment:
nit: for consistency can we use snake-case (`raw_id_token`)?

Reply:
whoops absolutely